src/annotator/hypothesis-injector.js

```diff
@@ -57,6 +57,18 @@ function hasHypothesis(iframe) {
   return iframeDocument.querySelector('script.js-hypothesis-config') !== null;
 }
 
+/**
+ * Remove the temporary configuration data added to a document by {@link injectClient}.
+ */
+export function removeTemporaryClientConfig(document_ = document) {
```

Comment:
Perhaps this function name could be clearer?

Reply:
I couldn't think of a name that I liked better this morning, but I did expand the comments a bit to mention the relation with `HypothesisInjector`.